### ISSUE TITLE
names that are too broad for their definitions (ciliary epithelial cells)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [obophenotype/cell-ontology#5]
```
TICKET: These terms have names that are too broad for their definitions, or 
definitions that are two narrow for their names:

id: CL:0002304
name: non-pigmented ciliary epithelial cell
def: "A multi-ciliated cell of the retina that lacks visual pigment and 
contributes to aqueous humor by secreting chloride ions. This cell type 
maintains gap junctions with pigmented epithelial cells." [GOC:tfm, 
PMID:15106942]

[Term]
id: CL:0002303
name: pigmented ciliary epithelial cell
def: "A ciliated epithelial cell of the retina, this cell type uptakes sodium 
chloride and passes it to non-pigmented ciliary epithelial cells." 
[PMID:15106942]
```

Original issue reported on code.google.com by `dosu...@gmail.com` on 27 Feb 2012 at 5:41

**Comment by paolaroncaglia** [comment]
@dosumis 

> ```
> TICKET: These terms have names that are too broad for their definitions, or 
> definitions that are two narrow for their names:
> 
> id: CL:0002304
> name: non-pigmented ciliary epithelial cell
> def: "A multi-ciliated cell of the retina that lacks visual pigment and 
> contributes to aqueous humor by secreting chloride ions. This cell type 
> maintains gap junctions with pigmented epithelial cells." [GOC:tfm, 
> PMID:15106942]
> 
> [Term]
> id: CL:0002303
> name: pigmented ciliary epithelial cell
> def: "A ciliated epithelial cell of the retina, this cell type uptakes sodium 
> chloride and passes it to non-pigmented ciliary epithelial cells." 
> [PMID:15106942]
> ```
> 
> Original issue reported on code.google.com by `dosu...@gmail.com` on 27 Feb 2012 at 5:41

Would you be ok with
- [ ] renaming CL:0002304 'non-pigmented ciliary epithelial cell' as 'retinal multi-ciliated non-pigmented epithelial cell'
- [ ] adding part_of 'retina'
- [ ] editing CL:0002303 'pigmented ciliary epithelial cell' so it's a subclass of 'ciliated epithelial cell' (rather than 'multi-ciliated epithelial cell'; rename it as 'retinal pigmented ciliary epithelial cell'; make it part_of retina
?
Also,
- [ ] fix typo in def of CL:0005012 'multi-ciliated epithelial cell' "A ciliated epithelial cell with many cilium." (cilium => cilia)

**Comment by bvarner-ebi** [comment]
@dosumis, unless you have other feedback, I will proceed with Paola's suggestions above.

**Comment by bvarner-ebi** [comment]
After taking a closer look at this, it appears the inconsistency in the original definitions is causing confusion rather than this being a narrow vs broad issue.

A 'non-pigmented ciliary epithelial cell', as it is referenced in the dbxref annotation, is not a retinal cell. It is part of the [ciliary body](http://purl.obolibrary.org/obo/UBERON_0001775), which is part of the uvea. These cells do contribute to production of the aqueous humour, but I have not found a reference that describes them as having cilia or being "multi-ciliated".

Similarly, a 'pigmented ciliary epithelial cell' is part of the ciliary body, not part of the retina. They do participate in the production of aqueous humour with the non-pigmented ciliary epithelial cells, but I have not found any reference that describes them as having cilia.

Proposed solution to come....

**Comment by bvarner-ebi** [comment]
Additionally:
[ ] fix typo in def of CL:0005012 'multi-ciliated epithelial cell' "A ciliated epithelial cell with many cilium." (cilium => cilia)
